tests/integration/test_wazuh_db/data/agent_messages.yaml

```diff
@@ -1,77 +1,199 @@
-  -
-    input: 'agent 000 vuln_cve clear'
-    output: "ok"
-    stage: "agent vuln_cve clear table"
+    input: 'agent 000 vuln_cves insert {"name":"test_package",
+                                        "version":"1.0",
+                                        "architecture":"x86",
+                                        "cve":"CVE-2021-1001",
+                                        "reference":"03c06c4f118618400772367b1cf7e73ce0178e02",
+                                        "type":"PACKAGE",
+                                        "status":"VALID",
+                                        "check_pkg_existence":true}'
+    output: 'ok {"action":"INSERT","status":"PKG_NOT_FOUND"}'
+    stage: "agent vuln_cves insert test_package when it does not exist in sys_programs"
+  -
+    input: 'agent 000 vuln_cves insert {"name":"test_package",
+                                        "version":"1.0",
+                                        "architecture":"x86",
+                                        "cve":"CVE-2021-1001",
+                                        "reference":"03c06c4f118618400772367b1cf7e73ce0178e02",
+                                        "type":"PACKAGE",
+                                        "status":"VALID",
+                                        "check_pkg_existence":false}'
+    output: 'ok {"action":"INSERT","status":"SUCCESS"}'
+    stage: "agent vuln_cves insert test_package without checking if the package is present in sys_programs"
   -
     input: 'agent 000 sql SELECT * FROM vuln_cves'
+    output: 'ok [{"name":"test_package","version":"1.0","architecture":"x86","cve":"CVE-2021-1001","reference":"03c06c4f118618400772367b1cf7e73ce0178e02","type":"PACKAGE","status":"VALID"}]'
+    stage: "agent vuln_cves checking test_package"
+  -
+    input: 'agent 000 vuln_cves insert {"name":"test_package",
+                                        "version":"1.0",
+                                        "architecture":"x86",
+                                        "cve":"CVE-2021-1001",
+                                        "reference":"03c06c4f118618400772367b1cf7e73ce0178e02",
+                                        "type":"PACKAGE",
+                                        "status":"VALID",
+                                        "check_pkg_existence":false}'
+    output: 'ok {"action":"UPDATE","status":"SUCCESS"}'
+    stage: "agent vuln_cves update already inserted entry"
+  -
+    input: 'agent 000 sql INSERT INTO sys_programs (scan_id,scan_time,format,name,priority,section,size,vendor,install_time,version,architecture,multiarch,source,description,location,triaged,cpe,msu_name,checksum,item_id)
+                                             VALUES(0,"2021/04/0722:00:00","deb","test package","optional","utils","7490","Core Bugman corebugman@wazuh.com",NULL,"1.0.0","amd64",NULL,NULL,"Test package",NULL,0,NULL,NULL,"e7dbc9bba5a0ee252866536225b952d3de7ea5cb","777fef8cc434b597769d102361af718d29ef72c1")'
```

Comment:
We should replace this Testing strings to something like:
Wazuh wazuh@wazuh.com

Reply:
Fixed!